### ISSUE TITLE
Error caused by commit 07a52e3f

### ISSUE THREAD

**Issue by nineKnight** [dracula/vim#93]
### What happened
```
Error detected while processing function <SNR>45_on_window_changed[17]..<SNR>45_init[20]..<SNR
>45_on_colorscheme_changed[10]..airline#load_theme[15]..airline#extensions#load_theme[1]..airl
ine#util#exec_funcrefs[2]..airline#extensions#tabline#load_theme[21]..airline#highlighter#exec
:
line   25:
E254: Cannot allocate color 0
Error detected while processing function <SNR>45_on_window_changed[17]..<SNR>45_init[20]..<SNR
>45_on_colorscheme_changed[10]..airline#load_theme[15]..airline#extensions#load_theme[1]..airl
ine#util#exec_funcrefs[2]..airline#extensions#tabline#load_theme[40]..airline#highlighter#exec
:
line   25:
E254: Cannot allocate color 0
Press ENTER or type command to continue
```

### What I expected to happen
```
No error
```

### Machine Info

- **Vim type (`vim`/`gvim`/`neovim`)**:vim
- **Vim version**:8.0.1766
- **OS**:Linux
- **Terminal/Terminal Emulator/VTE**:gnome
- **`TERM` environment variable**:xterm-256color

**Comment by benknoble** [comment]
@dsifford did this come up in your testing? Will try to repro later.

**Comment by benknoble** [comment]
Note this only occurs with tabline enabled.

**Comment by benknoble** [comment]
Also only occurs using Dracula airline theme

**Comment by benknoble** [comment]
Problem seems to be these lines:

```vim
  let g:airline#themes#dracula#palette.tabline = {
        \ 'airline_tabsel': s:clr('white', 'bg', 'bold'),
        \ 'airline_tabsel_right': s:clr('white', 'bg', 'bold'),
        \}
```

Still working to nail down what happens, but effectively these get turned into keys on the dictionary `g:airline#themes#dracula#palette.tabline` whose values are arrays. The arrays' first value should be a color (e.g. `#aabbcc`) but is actually becoming `0`.

**Comment by benknoble** [comment]
This only occurs for these lines because they use the value `'white'` in the first argument to `s:clr`—changing it to another color (e.g. `'green'`) works.

**Comment by benknoble** [comment]
This is because `s:clr` uses `g:dracula_palette.white` in this case, which doesn't exist!

**Comment by benknoble** [comment]
@nineKnight If you want to test #94 and confirm that it works, it's available to you.

**Comment by dsifford** [comment]
Whoops... Should have been `fg` and note `white`. My mistake

**Comment by benknoble** [comment]
Closed in 59fd603
